Formula/civl.rb

```diff
@@ -4,7 +4,8 @@ class Civl < Formula
   url "https://vsl.cis.udel.edu/lib/sw/civl/1.20/r5259/release/CIVL-1.20_5259.tgz"
   version "1.20-5259"
   sha256 "15bf63b3a92300e8432e95397284e29aaa5897e405db9fc2d56cd086f9e330d3"
-  revision 1
+  license all_of: ["GPL-3.0-or-later", "LGPL-3.0-or-later", "BSD-3-Clause"]
+  revision 2
```

Comment:
Sorry; should've realised this earlier: let's skip the revision bump.
```suggestion
  revision 1
```
Users don't need to reinstall this just to remove the `examples`.

Reply:
Yes but we will continue to ship the binaries, which means our CI will continue to fail?

Reply:
We can rebuild the bottles if you want. It won't affect CI even if we don't, though.

Reply:
Ok will probably re-bump here: #89642